source/preferences/manage-your-notifications.rst

```diff
@@ -103,7 +103,12 @@ On mobile, select the **Settings** |gear| icon and tap **Notifications**.
 
         **Customize notification keywords**
 
-        You can customize any additional non case-sensitive keywords to trigger notifications. For example, you can receive notifications for all messages and threads related to a specific topic, project name, or customer. Separate multiple keywords using commas or by pressing :kbd:`Tab`, and use :kbd:`Backspace` to manage keywords.
+        All Mattermost users can customize any additional non case-sensitive keywords to trigger notifications. For example, you can receive notifications for all messages and threads related to a specific topic, project name, or customer. Separate multiple keywords using commas or by pressing :kbd:`Tab`, and use :kbd:`Backspace` to manage keywords.
+
+        .. include:: ../_static/badges/ent-pro-only.rst
```

Comment:
@cwarnermm thank you for the PR, will this PR be attaching the screen shot mentioned here

Reply:
That image is already included in the codebase as one of our plan/deployment badges. Here's what that code renders as in the generated preview:
![image](https://github.com/mattermost/docs/assets/74422101/0a3f6208-b2d0-4eba-b090-c002df68af74)